src/api-requests/get-mongodb-tv-shows.ts

```diff
@@ -0,0 +1,43 @@
+import { MongoDBTVShow } from '../interfaces/mongodb-tv';
+import * as Sentry from '@sentry/nextjs';
+import { slugify } from '../service/build-content-items';
+
+export const getAllMongoDBTVShows = async (): Promise<MongoDBTVShow[]> => {
+    const apiKey = process.env.MONGODB_TV_API_KEY;
+    if (!apiKey) {
+        const e = new Error('MONGODB_TV_API_KEY is not defined.');
+        Sentry.captureException(e);
+        throw e;
+    }
+
+    try {
+        const req = await fetch(
+            'https://data.mongodb-api.com/app/mongodb-tv-app-bsvzg/endpoint/api/epg',
+            {
+                headers: {
+                    Accept: 'application/json',
+                    'Content-Type': 'application/json',
+                    apiKey,
+                },
+            }
+        );
+        const shows: MongoDBTVShow[] = await req.json();
+        // Don't want to include shows that have aired in the past for now.
+        const currentTime = new Date();
+        return shows.filter(show => new Date(show.till) > currentTime);
+    } catch (e) {
+        Sentry.captureException(e);
+        throw new Error('Failed to fetch MongoDB TV content.');
+    }
+};
+
+export const getMongoDBTVShowBySlug = async (
+    slug: string
+): Promise<MongoDBTVShow | null> => {
+    const allMongoDBTVShows = await getAllMongoDBTVShows();
```

Comment:
wont the API accept query param or something ?

Reply:
No the API is very simple, it just returns all of them.